hatch/src/cli/commands/new.rs

```diff
@@ -69,30 +69,25 @@ impl<'command> Command<'command> for New {
   }
 
   fn execute(&self, args: &ArgMatches<'command>) -> Result<Vec<Project>, HatchError> {
-    match fs::create_dir(self.project_path(args) + self.project_name(args).unwrap().as_str()) {
+    let name = self.project_name(args).unwrap();
+
+    match fs::create_dir(self.project_path(args) + &name.as_str()) {
       Err(e) => Err(HatchError::from(e)),
       Ok(_) => {
+        let path = self.project_path(args);
+        let version = self.project_version(args);
+        let kind = self.project_kind(args);
+
         let mut yaml_output = String::new();
-        
-        write!(&mut yaml_output, "name: {}\nversion: {}\nbuild: {}\n",
-               self.project_name(args).unwrap(),
-               self.project_version(args),
-               self.project_kind(args));
-
-        match fs::File::create(self.project_path(args)
-                               + self.project_name(args).unwrap().as_str()
-                               + "/Hatch.yml") {
+
+        write!(&mut yaml_output, "name: {}\nversion: {}\nbuild: {}\n", &name, &version, &kind);
+
+        match fs::File::create(self.project_path(args) + &name.as_str() + "/Hatch.yml") {
```

Comment:
Writing to a raw file in yaml syntax is good for now. Maybe if the yaml file grows more, we could adopt a yaml library for read/write.

Reply:
In the future I would like to have a focus on keeping the `Hatch.yml` file as simple as possible.  `yaml-rust` has support for 'emitting' yaml to a file; however, its somewhat undocumented.